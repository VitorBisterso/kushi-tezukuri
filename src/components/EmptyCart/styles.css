.emptyCart-container {
  display: flex;
  flex: 1;
  flex-direction: column;
  align-items: center;
  justify-content: center;
}

.emptyCart-container > img {
  width: 35vw;
}

.emptyCart-container p {
  font-family: 'Mali-Bold';
  font-size: var(--TEXT-BIGGEST);
  color: var(--COLOR-GOLD);
}

@media (max-width: 767px) {
  .emptyCart-container {
    margin-top: 5vw;
  }

  .emptyCart-container > img {
    width: 50vw;
  }

  .emptyCart-container p {
    font-size: var(--TEXT-MEDIUM);
  }
}
