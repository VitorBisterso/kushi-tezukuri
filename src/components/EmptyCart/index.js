import React from 'react'

import emptyCartImage from '../../assets/img/emptyCart.svg'

import './styles.css'

const EmptyCart = () => (
  <div className="emptyCart-container">
    <img src={emptyCartImage} alt="empty-cart" />
    <p>Seu carrinho est√° vazio...</p>
  </div>
)

export default EmptyCart
