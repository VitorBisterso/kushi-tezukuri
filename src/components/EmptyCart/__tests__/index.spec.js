import React from 'react'
import { shallow } from 'enzyme'

import EmptyCart from '../index'

describe('EmptyCart', () => {
  it('should render correctly', () => {
    const component = shallow(<EmptyCart />)

    expect(component).toMatchSnapshot()
  })
})
