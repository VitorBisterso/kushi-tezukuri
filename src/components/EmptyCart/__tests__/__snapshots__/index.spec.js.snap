// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyCart should render correctly 1`] = `
<div
  className="emptyCart-container"
>
  <img
    alt="empty-cart"
    src=""
  />
  <p>
    Seu carrinho est√° vazio...
  </p>
</div>
`;
